src/outfitMakerProofOfConcept

```diff
@@ -0,0 +1,82 @@
+import java.util.ArrayList;
+
+import components.set.Set;
+import components.set.Set1L;
+
+public class OutMakerProofOfConcept {
+    private String shirts;
+    private String pants;
+    private String shoes;
+    private ArrayList<OutfitMakerProofOfConcept> outfit;
+
+    public OutfitMakerProofOfConcept(String shirt, String pants, String shoes) {
+        this.shirt = shirt;
+        this.pants = pants;
+        this.shoes = shoes;
+        this.outfit = new ArrayList<>();
+    }
+
+
+    //kernel methods (simplified add and remove into one method + instead of has
+    //made them "getter" methods)
+
+
+    public String getShirt() {
+        return this.shirt;
+    }
+
+    public String getPants() {
+        return this.pants;
+    }
+
+        public String getShoes() {
+        return this.pants;
+    }
+
+
+
+    public void addOrRemoveItem(String type, String item, boolean isAdding) {
```

Comment:
This would be an excellent candidate for an enum. That way, you don't have to explicitly check the string.